.github/workflows/ngsast.yml

```diff
@@ -0,0 +1,49 @@
+# This workflow integrates ShiftLeft Inspect with GitHub
```

Comment:
I think this workflow file should go into another branch used by self-serve. The default master branch would not contain any workflow and we ingest the workflow files as part of a PR created by ShiftLeft.